config/routes.rb

```diff
@@ -11,6 +11,9 @@
   get "sources" => redirect('/quellen')
   resources :sources, path: 'quellen', only: [:index, :show]
 
+  get '/quelle_einreichen' => 'submit_source#new', as: :new_submit_source
+  post '/quelle_einreichen' => 'submit_source#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.